test_cases/labels.json

```diff
@@ -305,7 +305,7 @@
       "expected": {
         "properties": [
           {
-            "label": "Tórshavn, Denmark"
+            "label": "Tórshavn, Faroe Islands"
```

Comment:
nice, so we will be able to close https://github.com/pelias/pelias/issues/384, right?